float32 error_ball
float32 error_goal
float32 ball_distance
bool ball_seen
bool goal_seen
